tests/envs/dummy_env.py

```diff
@@ -0,0 +1,17 @@
+import gym
+
+
+class DummyEnv(gym.Env):
```

Comment:
A docstring/comment would be useful to state in which test this is used

Reply:
And maybe even renaming Dummy to something more descriptive for the specific test - it's actually crucially important that we don't accidentally create a DummyEnv in the main test file